tuftool/README.md

```diff
@@ -135,6 +135,24 @@ tuftool download \
 
 `tuftool` respects the `HTTPS_PROXY` and `NO_PROXY` environment variables.
 
+## Container
+
+You can build a simple container image to avoid needing to install the Rust toolchain and dependencies or your local machine.
+
+To build the image run use Docker or Finch (same argument syntax, just replace `finch` for `docker`):
+
+```shell
+finch build -t tuftool .
```

Comment:
We love finch, and it is great that you say "Docker of Finch" above, but Docker is the more common tool and is Finch is only for macOS. We should provide the command in the format that most people will use it.

(The cd makes it more clear where the dockerfile is).

```suggestion
cd finch
docker build -t tuftool .
```

Reply:
Fair enough, thought I'd try it.